packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/troubleshooting/index.md

```diff
@@ -6,13 +6,15 @@ After setting up an external service and an Event Hook or Inline Hook, you may n
 
 ### Preview tab ###
 
-A preview tab, accessible in the Developer Console, is available for the following two Inline Hooks:
+A preview tab, accessible in the Admin Console, is available for the following two Inline Hooks:
```

Comment:
@brianduffield-okta Can we link "preview tab" to the Inline Hooks Preview product documentation?
https://help.okta.com/en/prod/Content/Topics/automation-hooks/preview-inline-hooks.htm